### ISSUE TITLE
Accessibility issues: dropdown button missing aria-label

### ISSUE THREAD

**Issue by kikostadinov** [telerik/kendo-react#1287]
Reported by a client. 
Ticket: https://supportheroes.telerik.com/ticket/1562926

Color Picker dropdown button missing aria-label.

![screenshot](https://user-images.githubusercontent.com/27971306/165519670-1a97e813-fc5f-42d3-ac23-dedcf953ef34.png)
